contrib/openshift/README.md

```diff
@@ -1,50 +1,49 @@
 # OpenShift template for skydive agent
 
-This OpenShift template allows you to instantiate skydive in OpenShift.
+This OpenShift template allows you to instantiate skydive in OpenShift. 
 
-Assuming you work on project skydive:
+**Important**: For Skydive installation you need cluster-admin privileges. During the installation you have to grant same privileges to skydive.
 
-```
-oc new-project skydive
-```
-
-Install the template
+#### Create a new project with an empty node selector to asume that Skydive runs on all nodes. 
 
 ```
-oc create -f skydive-template.yaml
+oc adm new-project --node-selector='' skydive
 ```
 
-You need the DeploymentConfig to run in privileged:
+####  Skype analyser and agent need  extended  privileges
 
 ```
-oc adm policy add-scc-to-user privileged system:serviceaccount:skydive:default
+# analyser and agen run as privileged container
+oc adm policy add-scc-to-user privileged -z default
+# analyser need cluster-reader access get all informations from the cluster
+oc adm policy add-cluster-role-to-user cluster-reader -z default
 ```
 
-Instanciate the template:
-
-```
-oc new-app --template=skydive
-```
 
-Check that everything is working and created:
+####  Install from OpenShift template
 
 ```
-oc get pods
-oc get ds
+# adjust VERSION for the current version - for example: v0.20.1 or master
+VERSION=v0.20.1
+oc process -f https://raw.githubusercontent.com/skydive-project/skydive/${VERSION}/contrib/openshift/skydive-template.yaml | oc apply -f -
```

Comment:
@safchain if we want to point the yaml this way you will need to create a new tag.

Reply:
Or use ```VERSION=master```

Reply:
So let's use master for now because I don't know when we will release a new version.

Reply:
It means we can remove the VERSION completely? 
```suggestion
oc process -f https://raw.githubusercontent.com/skydive-project/skydive/master/contrib/openshift/skydive-template.yaml | oc apply -f -
```

Reply:
Thanks, changed.